tests/integration/test_orgs.py

```diff
@@ -101,9 +101,14 @@ def test_edit(self):
         self.auto_login()
         cassette_name = self.cassette_name('edit')
         with self.recorder.use_cassette(cassette_name, **self.betamax_kwargs):
-            o = self.get_organization()
+            o = self.get_organization(name='foo-org')
```

Comment:
This is because I had to record a new cassette using a test organization I have admin rights to.